patches/server/0943-Add-paper-dumplisteners-command.patch

```diff
@@ -93,12 +127,19 @@ index 0000000000000000000000000000000000000000..dab8d6fb489f8b3acc7c8fdaa1b5b6b8
 +
 +            sender.sendMessage(text("Listeners for " + args[0] + ":"));
 +
-+            Stream.of(handlers.getRegisteredListeners())
-+                .map(listener -> text(listener.getPlugin().getName(), GREEN)
++            for (final RegisteredListener listener : handlers.getRegisteredListeners()) {
++                final Component hoverText = text("Priority: " + listener.getPriority().name() + " (" + listener.getPriority().getSlot() + ")", WHITE)
++                    .append(newline())
++                    .append(text("Listener: " + listener))
```

Comment:
The things included by RegisteredListener#toString are already part of the (hover) message, so this shouldn't be needed.

Reply:
It's not needed anymore yes, however I kept it intentionally so that some basic info can be seen at a glance without hovering.

Reply:
Misread this earlier, it was meant to be `listener.getListener()`, fixed now